Formula/yaml.rb

```diff
@@ -0,0 +1,25 @@
+class Yaml < Formula
```

Comment:
The name is too generic.

Reply:
Hmm it's the name of my github project, and the name of the executable...

Reply:
That's beside the point.

Reply:
Ok well I can rename the class if you like...

YamlProcessor? YamlGo? YamlCli - though I suspect most formulas are cli tools...

Reply:
@ilovezfs ?

Reply:
ping @ilovezfs

Reply:
`yaml-processor` as formula name (and thus `YamlProcessor` as class name)

Reply:
Another option might renaming your project to `yq` or `yquery` since your description says you're the yaml version of jq. Just a thought.

Reply:
You know what that's actually kind of temping